tests/e2e/performance/test_small_file_workload.py

```diff
@@ -437,7 +510,7 @@ def test_smallfile_workload(self, ripsaw, file_size, files, threads,
 
         # wait for benchmark pods to get created - takes a while
         for bench_pod in TimeoutSampler(
-            120, 3, get_pod_name_by_pattern, 'smallfile-client', 'my-ripsaw'
+            240, 10, get_pod_name_by_pattern, 'smallfile-client', 'my-ripsaw'
```

Comment:
pls use constants for 'my-ripsaw' https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L164

Reply:
Change it.